lib/rails_stats/stats_calculator.rb

```diff
@@ -133,5 +136,47 @@ def calculate_tests
         @statistics.each { |k, v| @test_loc += v.code_lines if v.test }
         @test_loc
       end
+
+      def calculate_create_table_calls
+        if @schema_path && File.exist?(@schema_path)
+          count_create_table_calls(schema_path, "create_table")
+        elsif @structure_path && File.exist?(@structure_path)
+          count_create_table_calls(structure_path, "CREATE TABLE")
+        else
+          0
+        end
+      end
+
+      def count_create_table_calls(file_path, keyword)
+        create_table_count = 0
+        File.foreach(file_path) do |line|
+          create_table_count += 1 if line.strip.start_with?(keyword)
+        end
+        create_table_count
+      end
+
+      def calculate_sti
+        @sti = 0
+        Dir.glob(File.join(@root_directory, "app", "models", "*.rb")).each do |file|
+          File.foreach(file) do |line|
+            if line =~ /class\s+(\w+)\s*<\s*(\w+)/ && !(line =~ /class\s+(\w+)\s*<\s*(ActiveRecord::Base|ApplicationRecord)/)
```

Comment:
@hmdros I think this is too optimistic and it will generate false positives in certain applications. 

Many projects use app/models as a place to put models that don't necessarily extend an ActiveRecord::Base|ApplicationRecord descendant. 

I see that in your test you have this example: 

```
class Pets < User
end
```

That's certainly an STI. 

But what if someone has this in their app/models directory: 

```
class Pets < User
end
```

And then: 

```
class User 
...
end
```

That is not an STI and it will be counted as one (or two?)